### PR TITLE
Handle reboot message

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(2) ChrOS: support reboot](https://trello.com/c/LRnRpXeL/4100-2-chros-support-reboot)

File: test/unit/reboot-scheduler.js [hunk 1]
```diff
@@ -71,4 +71,21 @@ describe('Reboot Scheduler', () => {
...
+
+  it('should not reboot now when not in kiosk mode', () => {
+    sandbox.stub(envVars, 'isKioskSession').returns(false);
+
+    rebootScheduler.rebootNow();
+
+    sinon.assert.notCalled(chrome.runtime.restart);
+  });
+
+  it('should not reboot now successfully', () => {
```
Comment (rudibravo) [inline test/unit/reboot-scheduler.js]: I think it meant to say `should reboot now successfully`

Comment (andrecardoso) [reply test/unit/reboot-scheduler.js]: Correct. Thanks!

**Comment by tejohnso** [comment]
Shouldn't the reboot come from messaging service?

**Comment by andrecardoso** [comment]
I think it is coming from MS here: https://github.com/Rise-Vision/chrome-os-player/pull/66/files#diff-113ff46cf15064abc1c127f2a79f9c14R24

**Comment by tejohnso** [comment]
Hmm...I don't think it should be in viewer.js. It has nothing to do with viewer. And we want to make sure we retain the functionality when we get rid of viewer.

**Comment by andrecardoso** [comment]
@tejohnso makes sense. I think the problem is the name of that file. I think we can rename it to `player.js` as it is the player window and most of its code is not viewer related.

**Pull request merged**